Gem/Code/Source/AutoGen/NetworkTeleportCompatibleComponent.AutoComponent.xml

```diff
@@ -11,14 +11,29 @@
 	<ComponentRelation Constraint="Required" HasController="true" Name="NetworkTransformComponent" 
                        Namespace="Multiplayer" Include="Multiplayer/Components/NetworkTransformComponent.h" />
 
+    <Include File="Source/Effects/GameEffect.h" />
+
+    <ArchetypeProperty Type="GameEffect"  Name="TeleportEffect"  Init="" Container="Object" Count="1" ExposeToEditor="true" Description="Specifies the effect to play upon teleportation" />
+
     <RemoteProcedure
         Name="Teleport" 
         InvokeFrom="Server" 
         HandleOn="Authority" 
         IsPublic="true" 
         IsReliable="true" 
         GenerateEventBindings="true"
-        Description="Teleport occured RPC">
+        Description="Teleport occurred RPC">
+        <Param Type="AZ::Vector3" Name="TeleportedLocation"/>
+    </RemoteProcedure>
+
+    <RemoteProcedure
+        Name="NotifyTeleport"
+        InvokeFrom="Authority"
+        HandleOn="Client"
+        IsPublic="false"
+        IsReliable="false"
```

Comment:
IsReliable = true? I'd think we'd always want this to pop since teleport a pretty rare/significant event. Or we leaving this false since it doesn't really affect gameplay?

Reply:
Yeah, it's only a notification to other players (spectators) to spawn an effect.. if it comes through late it would look weird anyways.